test/testutil.h

```diff
@@ -335,3 +350,22 @@ void test_info_c90(const char *desc, ...) PRINTF_FORMAT(1, 2);
         }                                       \
     } while (0)
 #endif                          /* HEADER_TESTUTIL_H */
+
+
+/*
+ * The basic I/O functions used by the test framework.  These can be
+ * shimed when needed.  If shimed, they must ALL be replaced.
```

Comment:
shimmed.
Or maybe "overriden when needed.  Note that if one is, then all must be."